tutorials/publish/tutorial39.m

```diff
@@ -0,0 +1,187 @@
+%% Description
+% This tutorial fits a spinwave model to simulated powder data.
+
+%% Define instrument resolution parameters for MARI
+% This is a simplified parameterisation of the resolution of a
+% Time-of-Flight spectrometer which will be used to broaden the simulated
+% data. These parameters will depend on the instrument used - the instrument
+% contact should be able to provide these.
+
+Ei = 5; % meV
+
+% Resolution (from PyChop)
+eres = @(en) 512.17*sqrt((Ei-en).^3 .* ( 8.26326e-10*(0.169+0.4*(Ei./(Ei-en)).^1.5).^2 + 2.81618e-11*(1.169+0.4*(Ei./(Ei-en)).^1.5).^2));
+% Q-resolution (parameters for MARI)
```

Comment:
Ah actually these are parameters for MERLIN.

Sorry I think I put in the wrong instrument name when I wrote the original example.